PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -69,32 +69,30 @@ public  async Task<IActionResult>  UpdateAsync([FromRoute] int id, [FromBody] Bi
             return BadRequest("Validation error.");
         }
 
-        var birthday = await _dbcontext.Birthdays.FirstOrDefaultAsync(b => b.Id == id);
-        if (birthday == null)
+       
+        try
+        {
+            await _birthdayService.UpdateAsync(id, input);
+            return NoContent();
+        }
+        catch (InvalidOperationException)
         {
             return NotFound($"Birthday with Id={id} not found.");
         }
-        
-        birthday.UserFullName = input.UserFullName;
-        birthday.Date = input.Date;
-       await _dbcontext.SaveChangesAsync();
-        
-        return Ok(birthday);
     }
 
     [HttpDelete]
     [Route("{id:int}")]
     public async Task<IActionResult> DeleteAsync([FromRoute] int id)
     {
-        var birthday = await _dbcontext.Birthdays.FirstOrDefaultAsync(b => b.Id == id);
-        if (birthday == null)
+        try
+        {
+            await _birthdayService.DeleteByIdAsync(id);
+            return NoContent();
+        }
+        catch (InvalidOperationException)
         {
             return NotFound($"Birthday with Id={id} not found.");
         }
-
-       _dbcontext.Birthdays.Remove(birthday);
-       await _dbcontext.SaveChangesAsync();
-
```

Comment:
In Delete method you have to just use method Service.DeleteByIdAsync(..) and `return Ok()` without any data in response.